cmd/oras/internal/errors/errors.go

```diff
@@ -159,25 +159,17 @@ func NewErrEmptyTagOrDigest(ref string, cmd *cobra.Command, needsTag bool) error
 }
 
 // CheckMutuallyExclusiveFlags checks if any mutually exclusive flags are used
-// at the same time, returns an error when detecting used exclusive flags. It
-// detects the first set of conflict and returns the error without checking
-// the rest of the flag sets.
-func CheckMutuallyExclusiveFlags(fs *pflag.FlagSet, exclusiveFlagSets ...[]string) error {
-	var checkConflict = func(exclusiveFlagSet []string) []string {
-		changedFlags := []string{}
-		for _, flagName := range exclusiveFlagSet {
-			if fs.Changed(flagName) {
-				changedFlags = append(changedFlags, fmt.Sprintf("--%s", flagName))
-			}
+// at the same time, returns an error when detecting used exclusive flags.
+func CheckMutuallyExclusiveFlags(fs *pflag.FlagSet, exclusiveFlagSet []string) error {
```

Comment:
nit: you may consider
```suggestion
func CheckMutuallyExclusiveFlags(fs *pflag.FlagSet, exclusiveFlagSet ...string) error {
```